.github/workflows/push_release.yml

```diff
@@ -0,0 +1,45 @@
+name: Push to Release Branch 
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_branch:
+        description: 'Branch to push to'
+        required: true
+
+env:
+  ACTIONS_PAT: ${{ secrets.ACTIONS_PAT }}
+
+jobs:
+  push-release-branch:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set up Git
+      run: |
+        git config --global user.name '${{ github.actor }}'
+        git config --global user.email '${{ github.actor }}@users.noreply.github.com'
+    - name: Fetch all branches
```

Comment:
@Raghul-M , As per my understanding from the code,  the current workflow is not fetching the branch from an upstream repository. It only interacts with the repository where the action is running